content/en/references/localstack-cli-manual.md

```diff
@@ -0,0 +1,260 @@
+
+{{< command >}}
+$ localstack update [OPTIONS] COMMAND [ARGS]...
+{{< /command >}}
+
+The commands available for the `update` command are:
+
+| Name             | Description                          |
+| ---------------- | ------------------------------------ |
+| `all`            | Updates all LocalStack components    |
+| `docker-images`  | Updates the LocalStack Docker images |
+| `localstack-cli` | Updates the LocalStack CLI           |
+
+### `config`
+
+The `config` command is used to inspect your LocalStack configuration.
+
+{{< command >}}
+$ localstack config [OPTIONS] COMMAND [ARGS]...
+{{< /command >}}
+
+The commands available for the `config` command are:
+
+| Name       | Description                                   |
+| ---------- | --------------------------------------------- |
+| `show`     | Shows the LocalStack configuration values     |
+| `validate` | Validates the LocalStack configuration values |
+
+### `start`
+
+The `start` command is used to start your LocalStack container.
+
+{{< command >}}
+$ localstack start [OPTIONS]
+{{< /command >}}
+
+The options available for the `start` command are:
+
+| Name, Shorthand    | Default | Description                                      |
+| ------------------ | ------- | ------------------------------------------------ |
+| `--detached`, `-d` |         | Run the LocalStack container in detached mode    |
+| `--docker`         |         | Start LocalStack in a Docker container (default) |
+| `--host`           |         | Start LocalStack directly on the host machine    |
+| `--no-banner`      |         | Disable the LocalStack banner                     |
+
+### `stop`
+
+The `stop` command is used to stop your LocalStack container.
+
+{{< command >}}
+$ localstack stop [OPTIONS]
+{{< /command >}}
+
+### `wait`
+
+The `wait` command is used to wait for your LocalStack container to be ready.
+
+{{< command >}}
+$ localstack wait [OPTIONS]
+{{< /command >}}
+
+The options available for the `wait` command are:
+
+| Name, Shorthand   | Default | Description                                           |
+| ----------------- | ------- | ----------------------------------------------------- |
+| `--timeout`, `-t` |         | Timeout in seconds to wait for LocalStack to be ready |
+
+### `status`
+
+The `status` command is used to check the status of your LocalStack container and print status information.
+
+{{< command >}}
+$ localstack status [OPTIONS] COMMAND [ARGS]...
+{{< /command >}}
+
+The commands available for the `status` command are:
+
+| Name       | Description                                                          |
+| ---------- | -------------------------------------------------------------------- |
+| `docker`   | Query information about the LocalStack Docker image and runtime      |
+| `services` | Query information about running services in the LocalStack container |
+
+### `daemons`
+
+The `daemons` command is used to manage LocalStack daemon processes.
```

Comment:
```suggestion
The `daemons` command is used to manage LocalStack daemon processes (e.g., required to start up EC2 virtual machines on the host).
```

On a side note, the `daemons` commands are currently not working (hence probably not really being used). We might want to consider re-activating them, or phasing them out with one of the upcoming releases. /cc @viren-nadkarni